DevEdu.API/DevEdu.API/Models/InputModels/NotificationAddInputModel.cs

```diff
@@ -1,12 +1,15 @@
 ﻿using System.ComponentModel.DataAnnotations;
+using static DevEdu.API.Common.ValidationMessage;
 
 namespace DevEdu.API.Models.InputModels
 {
     public class NotificationAddInputModel
     {
-        [Required]
+        [Required(ErrorMessage = UserIdRequired)]
         public int UserId { get; set; }
-        [Required]
+        [Required(ErrorMessage = TextRequired)]
         public string Text { get; set; }
+        [Required(ErrorMessage = RoleIdRequired)]
+        public int RoleId { get; set; }
```

Comment:
actually, only one of UserId and RoleId is required
I mean that we can pass UserId without RoleId 
and RoleId without UserId
so we need to remove [Required] from these properties